CHANGELOG.md

```diff
@@ -23,6 +23,75 @@ and this project adheres to \[Semantic Versioning\](<http://semver.org/spec/v2.0
 - No longer using `tox` for test running.
 - Switched to `ruff` from `pylint` for faster linting.
 
+**Notes for updating from v3.x.x to v4.x.x**
+
+**Configuring the parsers**
+
+The script for launching calculation should be updated. In particular, note that the new parser systems takes a
+different format for the `parser_settings`.
+
+Previously, this looks like:
+```python
+parser_settings = {
+        "add_energies": True,
+        "add_forces": True,
+        "add_stress": True,
+        "add_bands": True,
+        "electronic_step_energies": True,
+    }
+builder.settings = Dict({'parser_settings' : parser_settings})
+```
+
+Now, one should use:
+
+```python
+parser_settings = {
+'include_quantity': ['energies', 'forces', 'stress'],
+'electronic_step_energies': True
+}
+builder.settings = Dict({'parser_settings' : parser_settings})
+# Getting energies
+energies = node.outputs.arrays.get_array("energy_extrapolated")
+forces = node.outputs.misc['forces']
+```
+
```

Comment:
Some information like below would be helpful:

Previously `forces` was stored in `Array` node.  Now the last forces are stored in `misc` node. To obtain forces at every structure iteration, use `trajectory`.